data/hai missions.txt

```diff
@@ -473,3 +473,428 @@ mission "Unfettered returning home"
+
+			label wait
+			`	When the hubub has quieted down a little one of the unarmed Wanderers steps out of the croud and proclames (rather grandly) "Hai Eruk and Human <first>, you are to go before the Assembly to determine the validity of Hai Eruk's request of the Wanderers. Hai Eruk you will be kept under guard until such time it is deemed unnecessary." And with that most of the Wanderers fly off leaving just Ret, the two armed guards, Sayari and Eruk on your ship's landing pad.`
+			`	"What where you thinking!" Sayari says rounding on Eruk.`
+			`	"I needed someone to get me here and I  knew you woulden't agree to it so I asked Captin <last> to bring me here."`
+			choice
+				`	(keep out of it)`
+					goto "keeping out"
+				`	"Eruk said you agreed to it."`
+					goto sayari
+				`	"You lied to me!"`
+					goto eruk
+
+			label "keeping out"
+			`	"So instead you go behind my back, undermine our position and shater any work towards a closer relationship betwen the Hai and the Wanderers. You would jeopardise everything I have worked for to meet your ends?"`
+			`	"<first>, why did you bring him here?" You are thankful that she isn't shouting at you.`
+			`	"Eruk said you agreed to it."`
+
+			label sayari
+			`	"Did he now? Well I hope he's proud of the diplomatic incident he caused." Sayari says storming off.`
+				goto ret
+
+			label eruk
+			`	Eruk looks appolgetic and is about to speak before Sayari interupts. "You lied for your own gain? I thought you where brought up better than this!" Sayari says storming off.`
+
+			label ret
+			`	Ret looks surprised but recovers quickly "You should head over to the Assembly building," he says pointing at a building that looks vaguely like a giant birds nest, "I won't be [allowed, permitted] to stay so I need to find you an interpreter." Ret flies off as well.`
+			`	Eruk starts walking away from your ship and towards the Assembly building with the Wanderer guards following close behind.`
+			choice
+				`	(follow him)`
+					goto assembly
+				`	(try to comfort him)`
+
+			`	You hurry after him but when you catch up he simply angrily waves you away. You resign yourself to simply having to follow him.`
+
+			label assembly
+			`	When you reach the Assembly building you notice large numbers of Wanderers flying into it to through an open roof but you can't help but notice that there are no visible doors on the side of the building.`
+			`	Eruk sighs and goes to sit down on a nearby wall while you start to walk around the edge of the building looking for a door.`
+			`	After going nearly the entire way round the building you find Iktat Ret and another slightly smaller Wanderer standings by the only door you've seen so far. Eruk is also with them but he doesn't seem to be any happier than before.`
+			`	"<first>, this is my child Iktat Kri. [They?, He?, She?] will be able to [understand, interpret] for you."`
```

Comment:
The point is that the wanderers don't have words for male/female so it won't translate well. Eruk mentions this if you visit him. I could take it out and have [He?, She?] instead.